7_react/react1/week2/todo-list-app/src/TodoItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TodoItem = ({ id, description, deadline, done, toggleDone, deleteTask }) => {
```

Comment:
Just a nit-pick: 
The code is difficult to read if the lines are long. We can do this instead:
```js
const TodoItem = ({ 
  id, 
  description, 
  deadline,
  done, 
  toggleDone, 
  deleteTask, 
}) => {
```